src/index.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2023 Kapeta Inc.
+ * SPDX-License-Identifier: MIT
+ */
+
+import { format, Target, GeneratedAsset } from '@kapeta/codegen-target';
+import Path from 'path';
+import { exec } from '@kapeta/nodejs-process';
+
+export default class DockerTarget extends Target {
+    constructor(options?: any) {
+        super(options, Path.resolve(__dirname, '../'));
+    }
+
+    protected _postProcessCode(filename: string, code: string): string {
+        return format(filename, code);
+    }
+
+    async postprocess(targetDir: string, files: GeneratedAsset[]): Promise<void> {
+        const dockerFileChanged = files.some((file) => file.filename === 'Dockerfile');
+
+        if (dockerFileChanged) {
+            console.log('Running docker build in %s', targetDir);
+            const child = exec('docker build .', {
```

Comment:
I assume we want to build this with a tag, so we can run it otherwise it will be build with a hash as the 'name'

Reply:
We dont actually know the name at this point I think - but doing this will still cause it to "cache" the build - so when we do tag it (in local cluster service) - it'll be faster